recipe/meta.yaml

```diff
@@ -20,12 +22,13 @@ requirements:
   host:
     - python
     - pip
-    - cython >=0.28.2
-    - numpy
+    - cython >=0.29.16
+    - numpy ==1.15.4  # [py != 38]
```

Comment:
@xhochy can you check my work here? NumPy 1.17.3 is the first to support python 3.8.

Reply:
Seems that this will need some work for other platforms:

```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-aarch64: {'numpy==1.15.4'}
```

cc @odidev  if you have any suggestions.

Reply:
Actually we build Python 3.8 wheels with numpy 1.16 on conda-forge. https://github.com/conda-forge/pandas-feedstock/pull/85 

Also made a PR to ease numpy pinnings for maintainers: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/701

Reply:
> Seems that this will need some work for other platforms:
> 
> ```
> conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-aarch64: {'numpy==1.15.4'}
> ```
> 
> cc @odidev if you have any suggestions.

@TomAugspurger, 
Can we use `numpy>=1.16.3`?
**Note:** AArch64 and PowerPC architectures are supported from numpy version >= 1.16.3(reference [1.16.3](https://anaconda.org/conda-forge/numpy/files?version=1.16.3)).

Reply:
Uwe fixed it in https://github.com/conda-forge/pandas-feedstock/pull/85 if you want to check that.